sympy/core/expr.py

```diff
@@ -3619,7 +3616,7 @@ def _expand_hint_key(hint):
             use_hint = hints[hint]
             if use_hint:
                 hint = '_eval_expand_' + hint
-                expr, hit = Expr._expand_hint(expr, hint, deep=deep, **hints)
+                expr = Expr._expand_hint(expr, hint, deep=deep, **hints)[0]
```

Comment:
I consider this kind of thing to be defensive programming. If the function is always expected to return a sequence of length 2 then this asserts that it has exactly two items:
```python
>>> a, b = [1, 2, 3]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack (expected 2)
```
The variable name can also be used to help understand what the function returns even if the `hit` variable is unused.

Reply:
I was also considering this and, e.g., for the mp-function returning unused accuracy variables I did not change it. I see the point to some extent here as well. (And next comment.)